addons/product/views/product_template_views.xml

```diff
@@ -127,10 +127,10 @@
                 <field name="id"/>
                 <templates>
                     <div t-name="activity-box">
-                        <img t-att-src="activity_image('product.template', 'image_128', record.id.raw_value)" role="img" t-att-title="record.id.value" t-att-alt="record.id.value"/>
-                        <div>
+                        <img class="rounded-circle" t-att-src="activity_image('product.template', 'image_128', record.id.raw_value)" role="img" t-att-title="record.id.value" t-att-alt="record.id.value"/>
```

Comment:
I'm not sure that it is desirable to round the border for other records than partners/users.

In particular for product image, the whole image is important as a square.